PowerValidatedSolutions.psm1

```diff
@@ -15877,7 +15877,8 @@ Function Invoke-IomDeployment {
                                         Show-PowerValidatedSolutionsOutput -message "Preparing the NSX to $operationsProductName Integration"
                                         foreach ($sddcDomain in $allWorkloadDomains) {
                                             Show-PowerValidatedSolutionsOutput -message "Preparing the NSX to $operationsProductName Integration for Workload Domain ($($sddcDomain.name))"
-                                            $StatusMsg = Add-NsxtPrincipalIdentity -server $jsonInput.sddcManagerFqdn -user $jsonInput.sddcManagerUser -pass $jsonInput.sddcManagerPass -domain $sddcDomain.name -principalId ("svc-iom-" + $(($sddcDomain.nsxtCluster.vipFqdn).Split('.')[-0])) -role enterprise_admin -WarningAction SilentlyContinue -ErrorAction SilentlyContinue -WarningVariable WarnMsg -ErrorVariable ErrorMsg
+                                            $outputPath = Join-Path -Path $pwd -ChildPath "certificates\"
```

Comment:
Where does $pwd come from?

Reply:
$pwd is a powershell default variable for holding the current working directory. I have referred from the following link.

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.4

Reply:
I have concerns over this change as your hardcoding `certificates\` as being the folder always being used which may not be the case as this can be customer specific and your also then assuming that its contained within the current working folder